jamventProto/jamctrllib/JamCtrlGasMix.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * FILE: JamCtrlGasMix.cpp
+ * 
+ * LICENSE: This file is licensed under the GNU General Public License v3.0
+ * Permissions of this strong copyleft license are conditioned on making available complete 
```

Comment:
Good that we are adding header and copyright.  Just wonder if this is the right license for us.  Just a comment. Nothing to change.